### DISCUSSION TITLE
Official Documentation as Plugin

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6363] [Q&A]
Hi @Jermolene 

This has been discussed before but I could not find it!

While @btheado is actively working on documentation, what do you think if convert the docs into a plugin! or at least tag all those tiddlers with docs, so users can simply drag and drop docs to their own wikis!

**Comment by joshuafontany** [comment]
I sometimes use this in my tiddlywiki.info file. This will pull in all docs from the "tiddlywiki.com" and "tidldywiki.com/dev" editions into a "Help" edition that can be served locally via Node.js (When my multi-wiki experiments were the focus, this worked well to have a Help edition served by the wiki-server).
```
    "includeWikis": [
        "/c/tw/TiddlyWiki5/editions/dev",
        "/c/tw/TiddlyWiki5/editions/tw5.com"
    ],
    "config": {
        "retain-original-tiddler-path": "true"
    }
```
